modules/data.atmosphere/tests/testthat/test.met2CF.csv.R

```diff
@@ -1,16 +1,60 @@
 context("testing csv import using met2CF.csv")
-
-format <- list(orig  = c("Corrected.Date.Time", "Solar.Radiation..W.m2.", "Temp..C.",
-                      "PAR_.umol_m.2_s.1.", "PAR_.mol_m.2_h.1.", "RH....", "Wind.Speed..m.s.", 
-                      "VPD", "Rain..mm."),
-               units = c("mdy_hm", "W m-2","celsius",
-                         "umol m-2 s-1", "mol m-2 h-1", "%", "m s-1", "Pa","mm h-1"),
-               bety  = c("datetime", "solar_radiation", "airT", 
-                      "PAR", NA, "relative_humidity", "Wspd", NA, "precipitation_rate"),
-               skip  = 0,
-               unit.row = TRUE,
-               na.strings = NA) 
-# met2CF.csv(in.path = "data", in.file = "met2CF.csv.csv", outfolder = tempdir(), 
+format <- list(
+       header = 1,
+       time_zone = "GMT",
+       time.row = 1,
+       orig = c(
+              "Corrected Date/Time", "Solar Radiation (W/m2)", "Temp (C)",
+              "PAR_(umol_m-2_s-1)", "PAR_(mol_m-2_h-1)", "RH (%)", "Wind Speed (m/s)",
+              "VPD", "Rain (mm)"
+       ),
+       units = c(
+              "mdy_hm", "W m-2", "celsius",
+              "umol m-2 s-1", "mol m-2 h-1", "%", "m s-1", "Pa", "mm h-1"
+       ),
+       bety = c(
+              "datetime", "solar_radiation", "airT",
+              "PAR", NA, "relative_humidity", "Wspd", NA, "precipitation_rate"
+       ),
+       skip = 0,
+       unit.row = TRUE,
+       na.strings = NA,
+       vars = list(
+              input_name = c(
+                     "Corrected Date/Time", "Solar Radiation (W/m2)", "Temp (C)",
+                     "PAR_(umol_m-2_s-1)", "PAR_(mol_m-2_h-1)", "RH (%)", "Wind Speed (m/s)",
+                     "VPD", "Rain (mm)"
+              ),
+              input_units = c("", "", "", "", "", "", "", "", ""),
+              bety_name = c(
+                     "datetime", "solar_radiation", "airT",
+                     "PAR", NA, "relative_humidity", "Wspd", NA, "precipitation_rate"
+              ),
+              storage_type = c(
+                     "%m/%d/%y %H:%M", NA, NA, NA, NA, NA, NA, NA, NA
+              )
+       ),
+       lat <- 42 + 47 / 60 + 30 / 6000,
+       lon <- 76 + 7 / 60 + 20 / 6000
+)
+# met2CF.csv(in.path = "data", in.file = "met2CF.csv.csv", outfolder = tempdir(),
 #            format = format,
 #            lat = 42 + 47/60 + 30/6000,
 #            lon = 76 + 7/60 + 20/6000)
+
+
+in.path <- "data"
+in.file <- "test.met2CF.csv.csv"
+outfolder <- tempdir()
+start_date <- lubridate::mdy_hm("03/01/13 18:00")
```

Comment:
Better to use ymd_hm here for consistency

Reply:
ISO format implemented. Can you review this again?